python/cuml/ensemble/randomforest.pyx

```diff
@@ -154,6 +153,7 @@ cdef class RandomForest_impl():
         self.bootstrap = bootstrap
         self.verbose = verbose
         self.n_bins = n_bins
+        self. quantile_per_tree = quantile_per_tree
```

Comment:
@Salonijain27 Is it possible to have max_features same as sqrt() default like sklearn. Also would be nice to have similar option on python side as sklearn. Check here. https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html